modules/files.nix

```diff
@@ -111,6 +116,10 @@ in
           for sourcePath in "$@" ; do
             relativePath="''${sourcePath#$newGenFiles/}"
             targetPath="$HOME/$relativePath"
+            if [[ -e "$targetPath" && -n "$HOME_MANAGER_BACKUP_EXT" ]] ; then
```

Comment:
Hmm, wouldn't it be necessary to have `-e $targetPath && ! -L $targetPath`? I imagine that as written you get backups of managed links as well?

Reply:
I _think_ because this is in `newGenFiles` it's only called with links that need to be created. (You'd know better than I!) I'll add the condition, as a belt-and-braces thing, though.

Reply:
It will be called for each file whenever a new generation is created. So without the check it would create a backup of managed links.